docs/iceberg-rest-service.md

```diff
@@ -60,21 +71,21 @@ Gravitino provides a pluggable metrics store interface to store and delete Icebe
 | `gravitino.auxService.iceberg-rest.metricsQueueCapacity`   | The size of queue to store metrics temporally before storing to the persistent storage. Metrics will be dropped when queue is full. | 1000          | No       | 0.4.0         |
 
 
-### Iceberg catalog configuration
+### Gravitino Iceberg catalog backend configuration
 
 :::info
-The Gravitino Iceberg REST catalog service uses the memory catalog by default. You can specify Hive or JDBC catalog for production environments.
+The Gravitino Iceberg REST catalog service uses the memory catalog by default. You can specify using a Hive or JDBC catalog in a production environment.
 :::
 
-#### Hive catalog configuration
+#### Hive catalog backend configuration
```

Comment:
Is it better changing to "Hive backend configuration"?

Reply:
ok

Reply:
done